__test__/CarouselCardsRow.test.js

```diff
@@ -21,11 +21,15 @@ describe(`CarousleCardsRow`, () => {
   }
 
   test(`displays cards in slider mode`, () => {
-    const wrapper = shallow(<CarousleCardsRow {...props} />)
-    expect(wrapper.find(`#slide`)).toHaveLength(1)
+    const wrapper = shallow(<CarouselCardsRow {...props} />)
+    expect(wrapper.find(Slide)).toHaveLength(1)
```

Comment:
Way better. :+1:

Be aware, thogh, that since a couple of months ago we started adding [jest-enzyme](https://www.npmjs.com/package/jest-enzyme) to our projects and you can use some shortcut matchers such as:
- [`toContainExactlyOneMatchingElement`](https://www.npmjs.com/package/jest-enzyme#tocontainexactlyonematchingelement)
- [`toExist`](https://www.npmjs.com/package/jest-enzyme#toexist)

Try to use them to have more readable tests.